src/main/java/net/sf/jabref/model/strings/LatexToUnicodeAdapter.java

```diff
@@ -0,0 +1,18 @@
+package net.sf.jabref.model.strings;
+
+import java.text.Normalizer;
+import java.util.Objects;
+
+import com.github.tomtung.latex2unicode.LaTeX2Unicode;
+
+/**
+ * Adapter class for the latex2unicode lib. This is an alternative to our LatexToUnicode class
+ */
+public class LatexToUnicodeAdapter {
+
+    public static String format(String inField) {
```

Comment:
This is not possible, since we need the unicode conversion in the model to internally convert fields of `BibEntries`. If we just had the formatter, we would have a dependency from model to logic, which is forbidden in the architecture.